cmd/cluster-node-tuning-operator/main.go

```diff
@@ -169,11 +176,17 @@ func operatorRun() {
 	}
 	metrics.RegisterVersion(version.Version)
 
+	fg, err := setupFeatureGates(context.TODO(), restConfig, ntoNamespace)
+	if err != nil {
+		klog.Exitf("failed to setup feature gates")
+	}
+
 	if !config.InHyperShift() {
 		if err = (&paocontroller.PerformanceProfileReconciler{
-			Client:   mgr.GetClient(),
-			Scheme:   mgr.GetScheme(),
-			Recorder: mgr.GetEventRecorderFor("performance-profile-controller"),
+			Client:      mgr.GetClient(),
+			Scheme:      mgr.GetScheme(),
+			Recorder:    mgr.GetEventRecorderFor("performance-profile-controller"),
+			FeatureGate: fg,
```

Comment:
and how do we use this value?

Reply:
Specified in the PR and in the comment.
```
if fg.Enabled(<featureGateName>) {
do something
}
```
featureGateName are lister under the `openshift/api`
Here is the PR that adds the mixedCPUs feature gate: https://github.com/openshift/api/pull/1684

Reply:
right, silly me. So is going to be used in a followup PR. Works for me.

Reply:
> right, silly me. So is going to be used in a followup PR. Works for me.

Exactly